dist/pagination/ds4/pagination.css

```diff
@@ -29,7 +29,13 @@
   width: 17px;
 }
 .pagination__item {
+  background: none;
+  border: none;
+  cursor: pointer;
```

Comment:
Should we leave this to the browser to decide? Or are we trying to mimic a link in all cases?